process-manager.js

```diff
@@ -88,38 +93,46 @@ class ProcessManager {
 	release(process) {
 		process.release();
 	}
+
 	send() {
 		if (!this.processes.length) {
 			return Promise.resolve(this.receive.apply(this, arguments));
 		}
 
+		// Prevents the arguments object from leaking.
+		let args = [];
+		for (let i = 0; i < arguments.length; i++) {
+			args.push(arguments[i]);
+		}
```

Comment:
It is, since you're not passing a reference to the entire arguments object to  `Array.prototype.slice.apply`. This avoids passing any direct reference to the arguments object to any other function, which leads to leaks like this one.